packages/protocol/lib/compatibility/ignored-contracts-v9.ts

```diff
@@ -10,19 +10,18 @@ export const ignoredContractsV9 = [
   'StableTokenRegistry',
   'Reserve',
   'ReserveSpenderMultiSig',
+]
 
-  'SortedOracles' 
- // Note: Sorted Oracles is a Celo Core Contract
-  // but as it has also been modified and deployed but the Mento team.
-  // We currently need work to be able to upgrade it again:
-  // https://github.com/celo-org/celo-monorepo/issues/10435
+export const ignoredContractsV9Only = [
+  'SortedOracles'
+  // In CR9 the SortedOracles contract was deployed by Mento team, in CR10 we redeployed it ourselves
```

Comment:
Maybe a bit more accurate and with a bit more detail, something like "Between CR9 and CR10, a Mento upgrade [add which MU it was?] also upgraded SortedOracles. For the purposes of our compatibility tests, we use the Mento version of the contract, so that we're comparing the most recent pre-CR10 contracts with the CR10 versions."